data_validation/cli_tools.py

```diff
@@ -551,8 +551,13 @@ def _configure_column_parser(column_parser):
 
 
 def _configure_schema_parser(schema_parser):
-    """Configure arguments to run column level validations."""
+    """Configure arguments to run schema level validations."""
     _add_common_arguments(schema_parser)
+    schema_parser.add_argument(
+        "--exclusion-columns",
```

Comment:
Let's document this in our README - https://github.com/GoogleCloudPlatform/professional-services-data-validator#schema-validations